servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -51,6 +53,53 @@ $ oc ex config patch master-config.yaml.prepatch -p "$(cat master-config.patch)"
 $ /usr/local/bin/master-restart api && /usr/local/bin/master-restart controllers
 ```
 
+[[automatic-sidecar-injection]]
+=== Automatic sidecar injection
+Automatic injection of the sidecar is supported through the use of an annotation within your application's yaml file. The name of the annotation is `sidecar.istio.io/inject` and its value must be set to `true` for injection to occur.
+
+This example shows the annotation used within the sleep test application. The additional proxy related containers are included when you create the application in the OpenShift cluster.
+
+.source,yaml
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sleep
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: sleep
+    spec:
+      containers:
+      - name: sleep
+        image: tutum/curl
+        command: ["/bin/sleep","infinity"]
+        imagePullPolicy: IfNotPresent
+----
+
+[[manual-sidecar-injection]]
+=== Manual sidecar injection
+
+[NOTE]
+When you use manual sidecar injection, ensure that you have access to a running cluster so the correct configuration can be obtained from the istio-sidecar-injector configmap within the istio-system namespace.
+
+Manual injection of the sidecar is supported through the upstream `istioctl` command. To obtain the executable and deploy an application using manual injection you should:
+
+- Download the appropriate https://github.com/istio/istio/releases/tag/1.0.4[installation] for your OS
+- Unpack the installation into a directory and include the bin directory in your PATH
+
+Once installed you can inject the additional containers by executing the following command:
```

Comment:
This bit about additional containers is confusing, because we've been talking about sidecars up until now.  My first thought was, what containers are we injecting?  Would it be clearer if we said something like "You can inject the sidecar into your application by executing the following command."  or "You can update your existing applications manually to add the sidecar by executing the following command."

https://istio.io/docs/reference/commands/istioctl/#istioctl-kube-inject

Reply:
I agree, thinking some of the content in the note above could be moved to the introduction to clarify things... is there a preview of how this looks anywhere (a la Netlify)?

Reply:
@brian-avery, no preview yet.  We were trying to get the preview-bot configured for @geekspertise yesterday but there were some problems.